### ISSUE TITLE
The expand-routes section of the Table Syntax reference is confusing.

### ISSUE THREAD

**Issue by ddeaguiar** [pedestal/pedestal-docs#66]
It implies that using `expand-routes` in a service's route definition is ok but this should not be done in all cases. It will lead to an error when [`default-interceptors`](https://github.com/pedestal/pedestal/blob/master/service/src/io/pedestal/http.clj#L234) is called because route expansion will be attempted again since maps [implement](https://github.com/pedestal/pedestal/blob/master/route/src/io/pedestal/http/route.clj#L355) the `Expandable Routes` protocol.

I suspect that most folks knowingly or unknowingly use `default-interceptors`.

**Comment by mtnygard** [comment]
This should also be fixed in the hello world guide. It shows use of expand-routes, which will mislead people.

**Comment by ddeaguiar** [comment]
@mtnygard I'm closing this issue. The usage of `expand-routes` came up again today in Slack #pedestal and I ended up checking the [source](https://github.com/pedestal/pedestal/blob/master/service/src/io/pedestal/http.clj#L191-L193) for `default-interceptors`. If `::http/routes` is a seq of maps then route expansion is skipped. I'm not sure what problem I ran into that lead to this issue but it you don't get an error when using `expand-routes` with the table syntax and invoking `default-interceptors`.
